### ISSUE TITLE
kubeadm fails if swap is enabled on the nodes

### ISSUE THREAD

**Issue by raphaelpoumarede** [sassoftware/viya4-iac-k8s#18]
I had a failure of the kubadm init task because the swap was enabled on my K8s nodes.

`Failed to run kubelet" err="failed to run Kubelet: running with swap on is not supported, please disable swap>`

After running the code below (inspired from [there](https://germaniumhq.com/2019/02/14/2019-02-14-Disabling-Swap-for-Kubernetes-in-an-Ansible-Playbook/)) to disable the swap on the K8s nodes, I was able to have a sucessful kubeadm init task.

```sh
bash -c "cat << EOF > ~/viya4-iac-k8s/disable-swap.yaml
---
- hosts: k8s
  tasks:
  - name: Disable SWAP since kubernetes can't work with swap enabled (1/2)
    shell: |
      swapoff -a
  - name: Disable SWAP in fstab since kubernetes can't work with swap enabled (2/2)
    replace:
      path: /etc/fstab
      regexp: '^([^#].*?\sswap\s+sw\s+.*)$'
      replace: '# \1'
EOF"
cd ~/viya4-iac-k8s
ansible-playbook -i inventory disable-swap.yaml -b
```
We might want to add this task in the playbook.

**Comment by thpang** [comment]
This is a requirement but don't thinks it documented anywhere. Nice place. Next time try a PR ;)

**Comment by thpang** [comment]
Add link to PR that will fix this one.

**Comment by jarpat** [comment]
Closing this was resolved in PR #37 and is fixed in 2.0.0 and newer releases.
